package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^13.5.0",
     "prop-types": "^15.8.1",
     "react": "^18.1.0",
+    "react-data-table-component": "^7.5.2",
```

Comment:
Great job using this library 👏 👏 I like it 👍 😃

Reply:
@Sahar-AbdelSamad  thank you 👍🏽